pkgs/development/compilers/llvm/10/default.nix

```
@@ -6,15 +6,14 @@
 
 let
   release_version = "10.0.0";
-  candidate = "rc5";
-  version = "10.0.0${candidate}"; # differentiating these is important for rc's
+  version = release_version; # differentiating these is important for rc's
```

This comment is confusing now.

---

How would you write it? We need a comment to make clear, why we have two variables with the same value. I just kept it this way, because this is consistent with the other llvm versions.

---

I see. Never mind then. Or take my suggestion.
```suggestion
  version = release_version; # differentiating these (variables) is important for rc's
```
I'll look into the history when `11-rc1` comes along, anyway.

---

I should have had `version = "${release_version}${candidate}"`. Too late now :-)

---

Added it into the commit, it's a lot clearer what is meant this way. Thanks!